PackageBuilder/Build.cs

```diff
@@ -53,13 +53,22 @@ partial class Build : NukeBuild
             ? RootDirectory.Parent
             : RootDirectory.Parent / "template-package-listing";
 
+        private string _currentListingUrl;
         [Parameter("Path to existing index.json file, typically https://{owner}.github.io/{repo}/index.json")]
-        string CurrentListingUrl =>
-            $"https://{GitHubActions.RepositoryOwner}.github.io/{GitHubActions.Repository.Split('/')[1]}/{PackageListingPublishFilename}";
+        public string CurrentListingUrl
+        {
+            get => _currentListingUrl ?? 
+                $"https://{GitHubActions.RepositoryOwner}.github.io/{GitHubActions.Repository.Split('/')[1]}/{PackageListingPublishFilename}";
+
+            set => _currentListingUrl = value;
+        }
```

Comment:
This is just meant to be used by passing parameters to the `nuke` Target call directly, right?

Reply:
Yes, its aadditional parameter you can see used here

https://github.com/Happyrobot33/Happys-VRC-tools/blob/main/.github%2Fworkflows%2Fbuild-listing.yml#L57

Reply:
Ideally, I would actually like to switch it back to being determined entirely at runtime, but I cant seem to find a way currently to get the actual domain setup of the user